source/generate.py

```diff
@@ -136,27 +128,18 @@ def generate_multi_track_sequence(
         This means that the first instrument is generated with less bias than the next one, and so on.
 
         """
-        generate_features_dict = {
-            "model_identification": self.model.transformer.base_model.name_or_path,
-            "inst_list": inst_list,
-            "density_list": density_list,
-            "temperature": self.temperature,
-            "max_seq_length": self.max_length,
-            "generate_until": self.generate_until,
-        }
+        generate_features_dict = self.make_feature_dict(self, inst_list, density_list)
```

Comment:
feartures would refer to columns in a dataset, but this dictionary seems to be rather hyperparameters

Reply:
I see what you mean - Corrected -